content/02.main-text.md

```diff
@@ -20,8 +20,23 @@ Although we requested that some authors participate for their specific expertise
 `TODO: confirm "most"`
 To coordinate this effort, we developed a manuscript writing process using the Markdown language, the GitHub software development platform [@url:https://github.com/greenelab/deep-review/], and our new Manubot tool [@url:https://github.com/greenelab/manubot-rootstock; @url:https://github.com/greenelab/manubot] for automating manuscript generation.
 
+### Manubot
 
-### Collaborative writing and reviewing
+We developed Manubot, a system for writing scholarly manuscripts via GitHub.
+With Manubot, manuscripts are written as plain-text markdown files, which is well suited for version control using git.
+The markdown standard itself provides limited yet crucial formatting syntax, including the ability to embed images and format text via bold, italics, hyperlinks, headers, blockquotes, inline code, codeblocks, blockquotes, and numbered or bulleted lists.
```

Comment:
> We could skip some of the formatting options that are less relevant for manuscripts like inline code and codeblocks.

I think all of these formatting options are relevant and important to scholarly writing. They are just traditionally absent from manuscripts because Word does a poor job supporting them.

Reply:
Point taken